fastlane/Fastfile

```diff
@@ -23,9 +23,8 @@ platform :ios do
       code_coverage: true,
       slack_only_on_failure: true,
       skip_slack: ENV['PREVIOUS_BUILD_STATUS'] == 'error', # var defined in Bitrise step
-      output_types: 'json-compilation-database,html,junit',
```

Comment:
@imaginaris Should we keep this? Only saw it being used by oclint

Reply:
Hmm I think we don't need it. Danger uses output from slather, and I think the same for Sonar